src/ProjectService.Business/Commands/Project/FindProjectsCommand.cs

```diff
@@ -71,6 +73,8 @@ private async Task<List<DepartmentData>> GetDepartmentsThroughBroker(List<Guid>
 
         if (response.Message.IsSuccess)
         {
+          _logger.LogInformation("Departments were taken from the service.");
```

Comment:
Maybe "Departments were taken from the Company Service."?

Reply:
Done